src/api/login.ts

```diff
@@ -44,12 +45,17 @@ export function loginRequest(params: LoginRequestParamsType): Promise<LoginRespo
     debug(`Making request to ${BASE_API_URL + LOGIN_URL_PATH} with options:`)
     debug(JSON.stringify(requestOptions, null, 2))
     fetch(BASE_API_URL + LOGIN_URL_PATH, requestOptions)
+      .then((res) => {
+        if (!payload.oneTimePassword) return res
```

Comment:
I think this could be a break point because res is not a promise that is going to be result so the next then step maybe is not going to be triggered never??

Reply:
It should not be, but I'll test it out.

> Hello Daniel!!
> 
> First pull the last changes of master branch.
> Your changes break one login test
> Provide a new custom test for your new awesome feature please to ensure this lib always works
> After that I will merge your pull request for sure!
> 
> Thank you

Unfortunately tests do not work out of the box. What I still had to do to get them working:
 - npm i -g mocha ts-node
 - I had to install something else but unfortunately I forgot what it was.

Returning from within a .then() should always allow the chain to continue.

Once I got the tests working, unfortunately my account got blocked. Reason being that a one time password can be used only once, and the tests fire numerous times with the same OTP. To make this work, the tests should only log in once, and then reuse that session. For tests like the logout, it should then prompt for a new OTP.

For now, once I get my account reactivated, I will be suspending my own testing, and simply keep using my own fork, as that one does work for me.